padinfo/padinfo.py

```diff
@@ -71,6 +71,7 @@
 from padinfo.view.pic import PicViewState
 from padinfo.view.series_scroll import SeriesScrollViewState
 from padinfo.view.transforminfo import TransformInfoViewState
+from .boardgenerator import BoardGenerator
```

Comment:
this should be an explicit path, `from padinfo/boardgenerator import BoardGenerator`

although actually you should rename `boardgenerator` to `board_generator`

two notes about this:

1. In Pycharm to rename a file safely, right-click on it and go to Refactor --> Rename and then type the new name, and it will fix all instances of you having imported from it automatically (although with the `.boardgenerator` syntax I'm not 100% sure if it will detect this one tbh but do it like that anyway)
2. Ok so to import a file, in your code you write like, `board = BoardGenerator(message.upper())` and then `BoardGenerator` has a squiggly red underline on it right. So then you press `alt+Enter` underneath `BoardGenerator` and itll be like `Import BoardGenerator....` and you're like YES! and then you pick the one you want to import (sometimes there's multiple options, e.g. if you wrote your own plus you have installed a library that generates one, etc). Then, it'll auto-add the library to the imports, with the full path written out, and you never had to scroll to the top of the file to do it. Super op!
    * So, to fix the relative path import here, delete the import altogether, and then press alt+Enter on BoardGenerator, and use this keyboard shortcut.